### PR TITLE
Benchmark & cmd cleanups

### PR DESCRIPTION
**Pull request by tv42**

### CODE REVIEW

File: bench_test.go [hunk 1]
```diff
@@ -11,32 +11,48 @@ import (
 
 func BenchmarkBase58EncodeToString(b *testing.B) {
 	data := make([]byte, 8192)
+	data[0] = 0xff // without this, it's just skipping zero bytes
 	b.SetBytes(int64(len(data)))
 	for i := 0; i < b.N; i++ {
-		basen.Base58.EncodeToString(data)
+		x := basen.Base58.EncodeToString(data)
+		_ = x
```
Comment (cmars) [inline bench_test.go:18]: What's the purpose of this assignment?

File: bench_test.go [hunk 2]
```diff
@@ -11,32 +11,48 @@ import (
 
 func BenchmarkBase58EncodeToString(b *testing.B) {
 	data := make([]byte, 8192)
+	data[0] = 0xff // without this, it's just skipping zero bytes
```
Comment (cmars) [inline bench_test.go]: Also curious about the skipping.. what's the internal mechanism you're working around here?

File: bench_test.go (hunk 1 above)
Comment (tv42) [inline bench_test.go:18]: The Go devs have indicated that with that, the compiler is forced to produce a value for x, and then discard it, as opposed to optimizing the whole part out. I don't have a reference to point to at hand, sorry about that.

Comment (cmars) [inline bench_test.go:18]: This sounds like what you're referring to: https://github.com/golang/go/issues/5790

**Comment by cmars** [comment]
Sorry to leave this hanging so long.. I was undecided on the assignment thing and then lost track of this PR.

**Pull request merged**